Package.swift

```diff
@@ -27,7 +27,7 @@ let package = Package(
         .library(name: "XCTSpezi", targets: ["XCTSpezi"])
```

Comment:
Might make sense to also explore the library here as well, allowing people to transition to the new naming without any deprivations?

Reply:
Ups yes. Forgot about that. Great catch